### PR TITLE
Pin pelican version to fix page building.

### PR DESCRIPTION
**Pull request by heistermann**
Trying to fix Pelican issue on Travis by pinning Pelican to 3.7.1 as suggested e.g. [here](https://github.com/getpelican/pelican-plugins/issues/937).

### CODE REVIEW

**Comment by heistermann** [comment]
...and sorry for not doing things by the book (without PR) in commit db0789d8a6206291b57822f91c51b3ea619c9759 before...

**Comment by kmuehlbauer** [comment]
@heistermann Not sure, if this is the correct fix, there is no post processed. Could we try to move to python 3.7 first? See [here](https://github.com/wradlib/landing-page/blob/master/.travis.yml). We might need to tweak requirements. I'll check locally.

File: .travis.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ env:
   # openradar-bot
   - secure: "eZQF/n5xYbr10rqkLr/yoDBpdf+Bu9JW1unPorUgLt3fajtXBFHO2H49dLsLmRCaIPnW6iFLonvVApuZNo4vx/sjQPfw+2mvUkCIX7KN4Od9CibH6vrJeSZZSGeSomWNYa0rKpLT8T+i1iRz1DJO+uMrVQ3+A6HCxAJOvfceD+7PwSzRQvowa2YuH3TJ+zuWa3rKA82nruQU7VlaNXmVBzQLccJ3FiJg7+rJHMm9zZuXU0cTxLCFB7okHhQErL9uiXcPcsNG/0U8k8zZZvDxCqWlavXmaK8dO9pVQsVva4NVcrB56nOhADB2lGvLjPysfQ0Pg3Wln7qmSTUQXWsALCF2qsMiMlyuP7uRBRKuei40g3yyYYphlLZREl7lBp4wgvL0s9972Cldx8NiuYaJ8U6H7haMpVvc+dRx4JhPxyR0FK219ARJg/seHyWV3Jp4HWwDyB/CT6LH4Yq+0BrmF1GLpn9/ep13XQR+pZB4IsFHXMuJM4UCPc/Zy97maCM+ZGHkGf7tIQgZdNI8o0SErVVbV2x97Srmsqjx79/SV70WPpgADQfGI3Nna16dZEYANLtU0wZx0Zy5+vZUMWnkpLtbiH4J5plezfT57gxMEWt48PilNcEz4UAL6PXy7JzIkFKznRNzTRt93SVdz2wZ15kRV/wZkYiEtCRxR4o2XRk="
 python:
-- 2.7
+- 3.7
```
Comment (kmuehlbauer) [inline .travis.yml:14]:
```
python:
- 3.7
sudo: required
dist: xenial
```

Comment (heistermann) [reply .travis.yml:14]: Ah, I see.

**Comment by kmuehlbauer** [comment]
OK, let's now try the pinning to 3.7.1. Seems to work locally.

**Comment by kmuehlbauer** [comment]
Seems something is terribly broken... :disappointed:

**Comment by heistermann** [comment]
The errors on
```
AttributeError: 'Registry' object has no attribute 'keys'
```
can be ignored for now? Appears to be a `markdown` related [issue](https://github.com/airbnb/knowledge-repo/issues/452).

**Comment by kmuehlbauer** [comment]
The thing is no output is produced: ```Done: Processed 0 articles, 0 drafts, 0 pages and 0 hidden pages in 0.28 seconds.```

**Comment by heistermann** [comment]
Oops. Maybe pinning `markdown` to `<3`?

**Comment by kmuehlbauer** [comment]
Yes, locally I have it at 2.6.11. We might pin to 2.6 then.

**Comment by heistermann** [comment]
Better.

**Comment by heistermann** [comment]
But kind of ugly. Yet, as long as it works...

**Comment by kmuehlbauer** [comment]
@heistermann There is one issue with a link to opendata.md [here](https://openradarscience.org/2018/05/andopendata/).

**Review by kmuehlbauer — APPROVED** [review]
LGTM!

**Comment by heistermann** [comment]
Thanks for your help, @kmuehlbauer !!

**Comment by kmuehlbauer** [comment]
@heistermann You've been on the right track already :+1: Now we know, it's building on python 3.7. Not that bad. Need to keep in mind the pinning! 

It will take a while for the changes to appear on the github CDN.

**Pull request merged**